### ISSUE TITLE
Symbol headings with a custom theme

### ISSUE THREAD

**Issue by ZeroIntensity** [mkdocstrings/mkdocstrings#650]
I'm writing a custom MkDocs theme for my project and I can't seem to get symbol type headings to show up.  The `mkdocstrings` config looks like so:

```yml
plugins:
    - mkdocstrings:
          handlers:
              python:
                  paths: [src]
                  options:
                      show_root_heading: true
                      show_object_full_path: false
                      show_symbol_type_heading: true
                      show_symbol_type_toc: true
                      show_signature: true
                      seperate_signature: true
                      show_signature_annotations: true
                      signature_crossrefs: true
```

But, in my generated HTML, I don't see any element that could contain the type header. In a documentation that uses `mkdocs-material`, I see a `<code>` element that gets generated, but I don't see that anywhere in mine. Is there a setting that I missed?

**Comment by ZeroIntensity** [comment]
Apparently, they only show when building with `mkdocs build` and not `mkdocs serve`

**Comment by pawamoy** [comment]
Hey again @ZeroIntensity, sorry for the long delay. I've glanced at your repo and indeed I notice the same thing as you. I'm not sure to understand why the style sheet isn't applied, because it is correctly added to the final site, but looking at your base.html template, it looks like you're not using `extra_css` in any way. I suppose using it will solve your issue :slightly_smiling_face: Closing, but feel free to comment further.
